docs/ecctl-getting-started.asciidoc

```diff
@@ -1,16 +1,21 @@
 [id="{p}-overview"]
 == Overview
 
-{p} is the CLI on steroids for the Elastic Cloud API. It wraps
-common operations commonly needed by Elastic Cloud operators within a
+NOTE: **{p} is still in beta and should be considered experimental!**
+
+{p} is the CLI on steroids for the Elastic Cloud Enterprise API. It wraps
+typical operations commonly needed by ECE operators within a
 single command line tool.
 
-Benfits of {p}:
+Benefits of {p}:
 
 * Easier to use for ops than using the Cloud UI or using the RESTful API directly
-* Help you automate the deployment lifecycle
+* Helps you automate the deployment lifecycle
 * Provides a foundation for integration with other tools, such as Terraform and Ansible
 
+Note that, right now, {p} targets ECE installations only. There are, however, plans
+to support public EC API in the future.
```

Comment:
Same comment as above. EC might be confusing would suggest to go with `Elasticsearch Service`